solrsearch-plugins/src/main/java/co.cask.hydrator.plugin/common/SolrOutputFormat.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.hydrator.plugin.common;
+
+import org.apache.hadoop.mapreduce.RecordWriter;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+import org.apache.hadoop.mapreduce.lib.output.NullOutputFormat;
+
+/**
+ * SolrOutputFormat - Output format class for Solr.
+ */
+public class SolrOutputFormat extends NullOutputFormat {
```

Comment:
implement ``OutputFormatProvider`` instead of extending ``NullOutputFormat``, look here for example,
https://github.com/caskdata/hydrator-plugins/blob/develop/cassandra-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/BatchCassandraSink.java#L183